config/locales/mailers.fr.yml

```
@@ -7,6 +7,8 @@ fr:
   mailers:
     common:
       hello: Bonjour,
+      farewell: "Belle journée, L'équipe RDV-Solidarités"
+      farewell_html: "<p>Belle journée,<br>L'équipe RDV-Solidarités</p>"
```

On n’envoie pas que des messages de bonne nouvelle, des fois les rdv sont liés à des situations personnelles difficiles. Souhaiter une belle journée peut être maladroit: je suis pour garder “À bientôt”, c’est neutre.
```suggestion
      farewell_html: "<p>À bientôt,<br>L’équipe RDV-Solidarités</p>"
```

(OK, je vais aussi vous faire une confidence: l’expression “belle journée” m’irrite autant que le mot “bienveillance”.)